lib/xml.js

```diff
@@ -73,22 +73,22 @@ var tag = function (name, prefix, value, attribs) {
 };
 
 var cdata = function(str) {
-  if (common.isString(str) && str.trim().length > 0) {
+  if (!_.isUndefined(str) && !_.isNull(str)) {
```

Comment:
Why did you update this line ?
Don't we wanna add a CDATA tag only for strings ?